lib/factory_bot/aliases.rb

```diff
@@ -10,7 +10,7 @@ class << self
 
   def self.aliases_for(attribute)
     aliases.map { |(pattern, replace)|
-      if pattern.match(attribute.to_s)
```

Comment:
Why are we removing `to_s`? Do we know why we were calling it here in the first place?

Reply:
I'm not sure about this part either, I tried it with symbol and string attributes and it both worked. But maybe there could be scenarios where attribute is another ruby object, that implements `to_s`? Don't know enough details about factory_bot there.

Reply:
Tracing back through the callers it seems like attribute here is always a symbol, and `Regexp#match` seems to work fine with a symbol (maybe that wasn't always true?), so change seems good to me.